tests/integration/targets/custom_image/tasks/tests.yml

```
@@ -0,0 +1,267 @@
+---
+# This test triggers a bug in the api
+#- name: Delete a non existent custom image by uuid
+#  cloudscale_ch.cloud.custom_image:
+#    uuid: "{{ item }}"
+#    state: absent
+#  loop:
+#    - '415caea5-da7c-4aaa-aaaa-ececd38fb8ea'
+
+- name: Fail delete a non existent custom image by name
+  cloudscale_ch.cloud.custom_image:
+    name: this-image-is-non-existent
+    state: absent
+  register: delete
+- name: Verify Fail delete a non existend custom image by name
+  assert:
+    that:
+      - delete is not changed
```

Shouldn't this directly test for failure?
```suggestion
      - delete is failed
```